oci/private/push.bzl

```diff
@@ -186,7 +186,7 @@ oci_push_lib = struct(
     attrs = _attrs,
     toolchains = [
         "@rules_oci//oci:crane_toolchain_type",
-        "@aspect_bazel_lib//lib:yq_toolchain_type",
+        "@aspect_bazel_lib//lib:jq_toolchain_type",
```

Comment:
I don't know how this got broken without CI getting red, but my `bazel test //...` invocation was red. since this isn't worth it's own PR, i'll slip this in here.